src/main/java/repo/OrderRepo.java

```diff
@@ -0,0 +1,28 @@
+package repo;
+
+import model.Order;
+import model.Product;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OrderRepo {
+    private Map<String, Order> orders = new HashMap<>();
+
+    public OrderRepo() {
+    }
+
+    public void add(Order order){
+        orders.put(order.getId(), order);
```

Comment:
Als User muss man im Parameter nur eine order übergeben - die ID wird hier extrahiert - sehr bequem in der Benutzung - klasse :relieved: